packages/terra-graphs/tests/wdio/Graphs/BarGraph/BasicBarGraphs/GroupedBars-spec.js

```diff
@@ -0,0 +1,12 @@
+Terra.describeViewports('Grouped bars', ['tiny', 'medium', 'large'], () => {
+  describe('Validates', () => {
+    before(() => browser.url('/raw/tests/terra-graphs/graphs/bar-graph/basic-bar-graphs/grouped-bars'));
+    it('canvas element', () => {
+      Terra.validates.screenshot('canvas', { selector: '.carbon-graph-canvas' });
+    });
+
+    it('legend element', () => {
+      Terra.validates.screenshot('legend', { selector: '.carbon-legend' });
```

Comment:
Since it is not taking any action between taking the first and second screenshots, it would be better to use a different (parent) selector that can capture both the canvas and legend elements in one bigger screenshot instead of two separate screenshots.

Reply:
Made those into one [test](https://github.com/cerner/terra-graphs/pull/106/commits/8f1b7d01eb1f006f648f922b7b0ca25095b52fb2#diff-9607c5a1555698426ee148b5c37a2a748a3849d6d3c5ba3b9081b737a2b8cccfR3-R6).